src/Admin/Services/AccessControlService.cs

```diff
@@ -17,6 +21,18 @@ public class AccessControlService : IAccessControlService
         _globalSettings = globalSettings;
     }
 
+    public bool UserHasPermission(Permission permission)
+    {
+        if (_globalSettings.SelfHosted)
+            return true;
+
+        var userRole = GetUserRoleFromClaim();
+        if (string.IsNullOrEmpty(userRole) || !RolePermissionMapping.RolePermissions.ContainsKey(userRole))
+            return false;
```

Comment:
curly braces 😁

Reply:
Thank you! I've fixed this and some other instances